libwebauthn/src/proto/ctap2/model/credential_protection.rs

```diff
@@ -0,0 +1,22 @@
+use minicbor::{Decode, Encode};
```

Comment:
Is this file supposed to be part of this PR? 
The `CredentialProtectionPolicy` isn't used anywhere, it seems.